### DISCUSSION TITLE
Does Flux2, e.g. image automation, support "Dismissing a pull request review" while master branch is protected?

### DISCUSSION THREAD

**Discussion by brant4test** [fluxcd/flux2#2784] [Q&A] [answered]
Using [protected branches](https://help.github.com/articles/about-protected-branches/) to block merging when pull requests have reviews that request changes help the team maintain quality, bug-free code.

Does Flux2 e.g. image automation, support [Dismissing a pull request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review) while the master branch is protected? 

Or any other suggestions? Our goal is - _the PR process is only for individuals while the Flux2 robot account skips the PR process._

Thanks!

**Comment by kingdonb** [comment, marked as answer]
In GitHub, and likely others, I think the only way to do what you're asking is to make the Flux robot account an admin user for the repository.

It is possible to configure branch protection to allow admins to bypass it, or it can be configured in fully-enforcing mode. You may have to reconfigure your branch protection rule to make this possible.
